composer.json

```diff
@@ -10,5 +10,10 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5"
+    },
+    "scripts": {
+        "test": [
+            "./vendor/bin/phpunit -c phpunit.xml --colors=always --stop-on-failure --coverage-text --coverage-clover coverage.xml --coverage-html ./coverage"
```

Comment:
This is fine for those who want to run tests locally. For me it was good enough to just rebuild container if I wanted to run tests against changed code.